name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/pdf/DABPDFExctractor.java

```diff
@@ -173,61 +177,36 @@ private void addSellTransaction()
                             String currency = asCurrencyCode(v.get("currency"));
                             if (currency.equals(t.getAccountTransaction().getCurrencyCode()))
                             {
-                                t.getPortfolioTransaction().addUnit(
-                                                new Unit(Unit.Type.FEE, Money.of(currency, asAmount(v.get("fees")))));
+                                t.getPortfolioTransaction().addUnit(new Unit(Unit.Type.FEE, Money.of(currency, asAmount(v.get("fees")))));
                             }
                         })
 
-                        .section("tax", "currency", "label").optional()
-                        .match("^(?<label>.*)Kapitalertragsteuer (?<currency>\\w{3}+) (?<tax>[\\d.]+,\\d+)-?$")
-                        .assign(this::addSellTaxUnit)
-
-                        .section("tax", "currency", "label").optional()
-                        .match("^(?<label>.*)Solidaritätszuschlag (?<currency>\\w{3}+) (?<tax>[\\d.]+,\\d+)-?$")
-                        .assign(this::addSellTaxUnit)
-
-                        .section("tax", "currency", "label").optional()
-                        .match("^(?<label>.*)Kirchensteuer (?<currency>\\w{3}+) (?<tax>[\\d.]+,\\d+)-?$") //
-                        .assign(this::addSellTaxUnit)
-
                         .wrap(t -> {
                             if (t.getPortfolioTransaction().getAmount() == 0L)
                                 throw new IllegalArgumentException("No amount found");
 
                             return new BuySellEntryItem(t);
-                        }));
-    }
-
-    @SuppressWarnings("nls")
-    private void addSellTaxUnit(BuySellEntry t, Map<String, String> v)
-    {
-        if (v.get("label").contains("im laufenden Jahr einbehaltene"))
-            return;
-
-        String currency = asCurrencyCode(v.get("currency"));
+                        });
 
-        // FIXME forex fees must update gross value
-        if (currency.equals(t.getAccountTransaction().getCurrencyCode()))
-            t.getPortfolioTransaction().addUnit(new Unit(Unit.Type.TAX, Money.of(currency, asAmount(v.get("tax")))));
+        addTaxesSectionsTransaction(type, pdfTransaction);
     }
 
-    @SuppressWarnings("nls")
```

Comment:
Just FYI, ich habe die SuppressWarning wieder reingenommen. Ansonsten bekomme ich in der Eclipse IDE ganz vielen Meldungen über nicht externalisierte Strings. (In den Eclipse Einstellungen habe ich das als Warnings eingestellt).